util.go

```diff
@@ -15,3 +15,13 @@ func SnowflakeTimestamp(ID string) (t time.Time, err error) {
 	t = time.Unix(0, timestamp*1000000)
 	return
 }
+
+//PermissionsSynced Checks if channel has permissions synced with another channel
+func PermissionsSynced(c1 *Channel, c2 *Channel) bool {
```

Comment:
I am not sure if this is wanted or not?

Reply:
I think it'd be best to leave this out; two channels being synced doesn't have a strong definition in the API; the `PermissionOverwrites` could have different orders, and without robust tests for the function I think that it'd be best to defer to the "lower-level" promise of DiscordGo and not include it.

Reply:
Okay

Reply:
One last thing, i added this for the https://i.nuuls.com/tVWo9.png